### DISCUSSION TITLE
Keystroke for redo?

### DISCUSSION THREAD

**Discussion by osenvosem** [LazyVim/LazyVim#3867] [Q&A] [answered]
The standard keystroke for neovim redo `<C-r>` is remapped (to what by the way? When I use it on a symbol it inserts something strange) for some reason. While `u` for undo is working. Keystroke telescope search doesn't really help.

**Comment by folke** [comment, marked as answer]
`<c-r>` is not remapped and works as it should

**Reply by osenvosem** [reply]
Sorry, found the reason.
